rasa/cli/train.py

```diff
@@ -88,12 +88,33 @@ def train(args: argparse.Namespace) -> Optional[Text]:
         persist_nlu_training_data=args.persist_nlu_data,
         core_additional_arguments=extract_core_additional_arguments(args),
         nlu_additional_arguments=extract_nlu_additional_arguments(args),
+        model_to_finetune=_model_for_finetuning(args),
+        finetuning_epoch_fraction=args.epoch_fraction,
     )
 
 
+def _model_for_finetuning(args: argparse.Namespace) -> Optional[Text]:
+    if args.finetune == train_arguments.USE_LATEST_MODEL_FOR_FINE_TUNING:
+        # We use this constant to signal that the user specified `--finetune` but
+        # didn't provide a path to a model. In this case we try to load the latest
+        # model from the output directory (that's usually models/).
+        return args.out
+
+    return args.finetune
+
+
 def train_core(
     args: argparse.Namespace, train_path: Optional[Text] = None
 ) -> Optional[Text]:
+    """Trains a Rasa Core model only.
+
+    Args:
+        args: Command-line arguments to configure training.
+        train_path: Path were trained model but unzipped model should be stored.
```

Comment:
```suggestion
        train_path: Path where trained model but not unzipped model should be stored.
```

...if I understand correclty.